tests/product/tests/kube_rules.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass, astuple
```

Comment:
@oren-zohar, In order to separate data and tests, may be a good idea to create data folder under product/tests and to move kube_rules.py file to this folder.